README.md

```diff
@@ -76,6 +76,26 @@ Reporter consoleReporter = new ConsoleReporter("sourceName");
 Reporter compositeReporter = new CompositeReporter(directReporter, consoleReporter);
 
 /* Construct Wavefront opentracing Tracer composed of console and direct reporter */
-Tracer tracer = new WavefrontTracer.Builder().withReporter(compositeReporter).build();
+Tracer tracer = new WavefrontTracer.Builder().build(compositeReporter);
+```
+
+### Global Span Tags
+You can add metadata to opentracing spans using tags. The WavefrontTracer builder supports different methods to add those tags.
+```java
+/* Construct WavefrontTracer.Builder instance */
+WavefrontTracer.Builder builder = new WavefrontTracer.Builder();
+
+/* Add individual tag key value */
+builder.withGlobalTag("env", "Staging");
+
+/* Add a map of tags */
+builder.withGlobalTags(new HashMap<String, String>() {{ put("severity", "sev-1"); }});
+
+/* Add a map of multivalued tags since Wavefront supports repeated tags */
+builder.withGlobalMultiValuedTags(new HashMap<String, Collection<String>>() {{ 
```

Comment:
why mention adding applicationTags as global tags in ReadMe instead of `withApplicationTags(ApplicationTags applicationTags)` ?